smriprep/interfaces/surf.py

```diff
@@ -233,8 +259,31 @@ def fix_gifti_metadata(in_file: str, newpath: Optional[str] = None) -> str:
     if pointset.meta.get("GeometricType") == "Sphere":
         pointset.meta["GeometricType"] = "Spherical"
 
-    if newpath is not None:
+    if newpath is None:
         newpath = os.getcwd()
     out_file = os.path.join(newpath, os.path.basename(in_file))
     img.to_filename(out_file)
     return out_file
+
+
+def make_ribbon(
+    white_distvols: list[str],
+    pial_distvols: list[str],
+    newpath: Optional[str] = None,
+) -> str:
+    base_img = nb.load(white_distvols[0])
+    header = base_img.header
+    header.set_data_dtype("uint8")
+
+    ribbons = [
+        (np.array(nb.load(white).dataobj) > 0) & (np.array(nb.load(pial).dataobj) < 0)
+        for white, pial in zip(white_distvols, pial_distvols)
+    ]
+
+    if newpath is None:
+        newpath = os.getcwd()
+    out_file = os.path.join(newpath, "ribbon.nii.gz")
+
+    ribbon = base_img.__class__(ribbons[0] | ribbons[1], base_img.affine, header)
```

Comment:
small nitpick - can we remove operations when initializing a class, just to make debugging easier?

```suggestion
    ribbon_data = ribbons[0] | ribbons[1]
    ribbon = base_img.__class__(ribbon_data, base_img.affine, header)
```